### ISSUE TITLE
This one question doesn't make sense.

### ISSUE THREAD

**Issue by Helloyoufreak1235** [8values/8values.github.io#179]
There is a question that says "Even when protesting an authoritarian government, violence is not acceptable." This question pushes you more authoritarian if you agree with it and more libertarian if you disagree. That doesn't make sense because Totalitarian ideologies which as Fascism and Marxism-Leninism want a revolution against the current order. Anarchists such as Anarcho-Communists, Mutualists and Anarcho-Capitalists* also want a revolution. We can conclude from this that disagreeing with the statement would make you more extremist not specifically towards authoritarianism or libertarianism. Agreeing with the statement would make you more towards the Centre.

*Many Anarcho-Capitalists do not want a revolution.

**Comment by Khazbs** [comment]
I don't think this question is meant to take one's view of revolutions into account, otherwise the answer would also influence the scale of "traditionalism/progressivism". Maybe you could suggest a way to reword the question more clearly? If put in context of a revolution, as I see it, the question is more about how loyal the individual is to the TO BE state, rather than the AS IS state.

**Comment by olivertwistor** [comment]
The way I interpret the question is neither whether one supports a revolution nor how extremist vs. centrist one is. I interpret it as a question about law and order; whether it's ever okay to use force as a means to change (or to a lesser extent, break the laws if one believes them to be unjust).

I'm as progressive as they come (I think I got like 98-99 % on that scale), but I'm also centrist and strongly oppose political violence. In my view, progressivism has nothing to do with whether one accepts political violence against an authoritarian regime.
